guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -24,13 +24,13 @@ ifdef::foreman-el,katello[]
 
 include::proc_configuring-repositories-el.adoc[]
 
+ifdef::foreman-el[]
```

Comment:
where is this `ifdef` closed? the one in line 35 *technically* belongs to the `ifdef` in line 19?